### PR TITLE
WRLDBAEMS-87-config: Carousel

### PR DESCRIPTION
**Pull request by adobe-rpal**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://wrldbaems-87-config--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: component-definition.json [hunk 1]
```diff
@@ -602,6 +602,27 @@
...
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Carousel",
+                  "model": "carousel",
+                  "filter": "carousel",
+                  "item1": {
+                    "sling:resourceType": "core/franklin/components/block/v1/block/item",
+                    "name": "Cards",
+                    "model": "cards"
```
Comment (thomsebastin) [inline component-definition.json:620]: I think we already have cards by default. Can we rename this into something else and maybe singular name?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

File: component-definition.json (hunk 1 above)
Comment (adobe-rpal) [reply component-definition.json]: Fixed, added card items.

File: component-definition.json [hunk 2]
```diff
@@ -602,6 +602,29 @@
...
+                "template": {
+                  "name": "Carousel",
+                  "model": "carousel",
+                  "filter": "carousel",
+                  "item1": {
+                    "sling:resourceType": "core/franklin/components/block/v1/block/item",
+                    "name": "Card",
+                    "model": "card",
+                    "image": "/content/dam/world-bank/sample/placeholder-black.png",
+                    "text": "<h3>Lorem Ipsum</h3><p>Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum</p><p class = 'button-container'><a href='#'>Read Bio</a></p>"
```
Comment (thomsebastin) [inline component-definition.json:622]: Please add proper meaningful text that'll help the author.

Comment (adobe-rpal) [reply component-definition.json]: Done

**Review by mrugesh-adobe — APPROVED** [review]

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**